@xen-orchestra/lite/src/App/TabConsole.tsx

```diff
@@ -89,8 +89,8 @@ const TabConsole = withState<State, Props, Effects, Computed, ParentState, Paren
           <IntlMessage id='consoleNotAvailable' />
         </p>
       ) : (
-        <div  style={{ flex: 1, position: 'relative' }}>
-          <Console vmId={vmId} scale={state.consoleScale} setCtrlAltDel={effects.setCtrlAltDel}/>
+        <div style={{ flex: 1, height: '80%', position: 'relative' }}>
+          <Console vmId={vmId} scale={state.consoleScale} setCtrlAltDel={effects.setCtrlAltDel} />
         </div>
```

Comment:
```suggestion
        <div style={{ flex: 1, position: 'relative', overflow: 'hidden' }}>
          <div style={{height:'100%', width:'100%'}}>
            <Console vmId={vmId} scale={state.consoleScale} setCtrlAltDel={effects.setCtrlAltDel} />
          </div>
        </div>
```
* position relative to ensure the child use the size of the flex item to compute 100%
* overflow:hidden to ensure it try to fit the content 
* a child not flexed to ensure Console compute its size corectly 

what do you think of this ? it's more verbose, but I can at least understand why it works. I didn't find any solution without the additionnal div